HarmonyCore/FileIO/FileChannelManager.dbl

```diff
@@ -84,43 +326,98 @@ namespace Harmony.Core.FileIO
             fileName, @string
             openMode, FileOpenMode
         proc
+            data handle, D_HANDLE
             data channel, i4, 0
             try
             begin
+                if (RemoteFile(fileName) && connectionLoadCount > 1)
```

Comment:
Can you swap the order of this if test? RemoteFile is somewhat expensive because it involves string operations while checking for the connectionLoadCount is very fast.

Reply:
Perfectionist, LOVE it :)

Reply:
I guess is actually less a perf thing and more a desire to have a path through the code that behaves exactly like the old version, just in case there are bugs lurking in RemoteFile

Reply:
Done